core/GlobalState.cc

```diff
@@ -652,13 +652,21 @@ SymbolRef GlobalState::findRenamedSymbol(SymbolRef owner, SymbolRef sym) {
     SymbolData ownerScope = owner.dataAllowingNone(*this);
 
     if (nameData->kind == NameKind::UNIQUE) {
+        if (nameData->unique.uniqueNameKind != UniqueNameKind::MangleRename) {
+            return Symbols::noSymbol();
+        }
         if (nameData->unique.num == 1) {
             return Symbols::noSymbol();
         } else {
             ENFORCE(nameData->unique.num > 1);
             auto nm =
                 lookupNameUnique(UniqueNameKind::MangleRename, nameData->unique.original, nameData->unique.num - 1);
-            return nm.exists() ? ownerScope->members()[nm] : Symbols::noSymbol();
```

Comment:
I think this was using the `non-const` version of the `members()` method because this method was't const. I fixed that in another PR.